src/screens/CreateToken.js

```diff
@@ -151,23 +107,22 @@ class CreateToken extends React.Component {
     } else {
       address = this.refs.address.value;
     }
-    const promise = new Promise((resolve, reject) => {
-      const retPromise = hathorLib.tokens.createToken(hathorData.input, hathorData.output, address, this.refs.shortName.value, this.refs.symbol.value, parseInt(this.refs.amount.value*(10**hathorLib.constants.DECIMAL_PLACES), 10), this.state.pin);
-      retPromise.then((token) => {
-        // Update redux with added token
-        tokens.saveTokenRedux(token.uid);
-        resolve(token);
-      }, (message) => {
-        reject(message);
-      });
-    });
 
-    promise.then((token) => {
+    const retPromise = hathorLib.tokens.createToken(
+      address,
+      this.refs.shortName.value,
+      this.refs.symbol.value,
+      parseInt(this.state.amount*(10**hathorLib.constants.DECIMAL_PLACES), 10),
```

Comment:
We should move it to an utils method `valueFloatToInt(this.state.amount, hathorLib.constants.DECIMAL_PLACES)`.

Reply:
Done